docs/intro/contributing.txt

```diff
@@ -164,27 +232,20 @@ into the Django ``tests/`` directory and then running:
 
 .. code-block:: console
 
-    $ pip install -r requirements/py3.txt  # or py2.txt if you are running Python 2
+    $ pip install -r requirements/py3.txt
 
 Now we are ready to run the test suite. If you're using GNU/Linux, Mac OS X or
 some other flavor of Unix, run:
 
 .. code-block:: console
 
-    $ PYTHONPATH=.. ./runtests.py
+    $ ./runtests.py
 
 If you're on Windows, the above should work provided that you are using
```

Comment:
Maybe it would be worth introducing Git Bash from the start as that might help eliminate Windows differences.